src/pages/Group/Group.js

```diff
@@ -1,5 +1,314 @@
+                </div>
+            </div>
+        );
+    };
+
+    const priceRangeCards = () => {
+        return (
+            <div className='cuisine-row-container'>
+                {priceRanges.map((priceRange, index) => (
+                    <div className='small-individual-card'>
+                        <FilterButton content={
+                            <div>
+                                <p>{priceRange.amount}</p>
+                            </div>
+                            } 
+                        onClickHandler={() => handleSingleBtnColor(priceClicked, setPriceClicked, index)}
+                        onFilter={priceClicked[index]}
+                        />
+                    </div>
+                ))}
+            </div>
+        );
+    };
+
+    const allergensIndividualCards = allergens.map((allergen, index) => (
+        <div className='cuisine-individual-card'>
+            <FilterButton content={
+                <div>
+                    <p>{allergen.type}</p>
+                </div>
+                } 
+                onClickHandler={() => handleMultiClick(dietClicked, setDietClicked, index)}
+                onFilter={dietClicked[index]}
+            />
+        </div>
+    ));
+
+    const allergensCards = () => {
+        return (
+            <div className='cuisine-cards-container'>
+                <div className='cuisine-row-container'>
+                    {allergensIndividualCards.slice(0, allergensIndividualCards.length/2)}
+                </div>
+                <div className='cuisine-row-container'>
+                    {allergensIndividualCards.slice(allergensIndividualCards.length/2)}
+                </div>
+            </div>
+        );
+    };
+
+    const mealsIndividualCards = meals.map((meal, index) => (
+        <div className='cuisine-individual-card'>
+            <FilterButton content={
+                <div>
+                    <p>{meal.type}</p>
+                </div>
+                } 
+                onClickHandler={() => handleMultiClick(mealClicked, setMealclicked, index)}
+                onFilter={mealClicked[index]}
+                />
+        </div>
+    ));
+
+    const mealCards = () => {
+        return (
+            <div className='cuisine-cards-container'>
+                <div className='cuisine-row-container'>
+                    {mealsIndividualCards.slice(0, mealsIndividualCards.length/2)}
+                </div>
+                <div className='cuisine-row-container'>
+                    {mealsIndividualCards.slice(mealsIndividualCards.length/2)}
+                </div>
+            </div>
+        );
+    }
 
+    return (
+        <>
+            <Header />
+            <div className='group-hero'>
+                <h1 className='group-title'>Choose your preferences and create your group</h1>
+                <div className='group-cards'>
+                    <div className='left-card-container'>
```

Comment:
if you could rename all classes to follow the "group-<current classname>", that would be great. This is just to make sure that your styles don't affect any other parts of current code that may contain class names like "left-card-container". So for example, the "left-card-container" would be changed to 'group-left-card-container". Apply this change to all class names